### ISSUE TITLE
02.1.1 Linear Regression with Numpy

### ISSUE THREAD

**Issue by shangeth** [shangeth/OpenCV-DL-PyTorch#1]
https://github.com/shangeth/OpenCV-DL-PyTorch/blob/master/02-Intro-to-Deep-Learning/02.1-Linear-Regression/02_1_1_linear_regression.ipynb

- Add images/GIF to visualize gradient descent
- Markdown rendered for derivation of Gradient descent is different than colab.
